index.html

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>Document</title>
+</head>
+<body>
+    <div class="container">
+        <header>
+            <nav class="menu">
+                <h4 class="logo">My Logo</h4>        
+                <img src="Icon - Menu.png" alt="Icon-menu"></nav>
+        </header>
+        
+    
+        <main class="about">
+            <section class="intro">
+                <h2>I’m Karthika. <br>
+                    Glad to see you!</h2>
+                <p>I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience!
+                     If you like what you see and have a project you need coded, don’t hesitate to contact me.</p>
+            </section>
+            <section class="connect">
+                <h4>LET'S CONNECT</h4>
+            </section>
+            <section class="social">
+                
+                    <img src="Vector.png" alt="Twitter">
+                    <img src="Linkedin icon.png" alt="Linked in">
+                    <img src="msg.png" alt="message">
+                    <img src="github.png" alt="github">
+                    <img src="finger.png" alt="finger">            
+    
+                
+            </section>
```

Comment:
- [ ] It is important to make use of semantic tags as much as possible in our application and as such i will suggest rather than using the non-semantic `div` tag as container for the social media icons, you make use of the semantic `ul` tag and each link would be wrapped in an `li` tag as a list item. see the example below for more reference :+1: 
```html
<ul class="socials">
  <li>
    <a href="https://github.com/"><img src="./assets/github.png" alt="github"/></a>
  </li>
  <li>
    <a href="https://linkedin.com/in"><img src="./assets/linkedin.png" alt="linkedin"/></a>
  </li>
</ul>
```
- [ ] Ensure each icon is wrapped as a link which can be visited by users :+1: